packages/paste-website/src/pages/components/combobox/index.mdx

```diff
@@ -104,67 +106,96 @@ export const pageQuery = graphql`
 The Combobox is an opinionated implementation and wrapper around the **very excellent [Downshift component](https://downshift.netlify.app/downshift)**.
 
 Combobox allows a user to make a selection from a styled list box of options. Each option can consist of more than just text,
-_e.g._ text paired with an icon. It can also be set up with autocomplete/typeahead functionality so users can easily find a
+_e.g._ text paired with an icon. It can also be set up with autocomplete functionality so users can easily find a
 specific option.
 
+### What’s the difference between Select and Combobox?
+
+At its most basic, a Select has an options list that’s styled according to the browser default.
+A Combobox has a Twilio-styled options list and can allow for additional functionality like
+autocomplete and grouping options.
+
+Use a [Select](/components/select) when:
+
+- You need a native picker experience, especially on mobile devices.
+- Users will be selecting from a list of 4-10 options, or a sorted list of highly familiar options (e.g., alphabetical list of states or countries).
+- You need the component to work out-of-the-box across all operating systems and browsers.
+
+Use a [Combobox](/components/combobox) when:
+
+- You need a Twilio-styled options list.
+- You need to show more than text in an option (e.g., text paired with an icon).
+- You need to group options under labels.
+- You need to disable options in the list.
+- Users would benefit from autocomplete functionality (e.g., autocomplete, search). For example, autocomplete may be useful when users need to select from a list of more than 10 options.
+- You need to lazy load a much longer list of options to improve page load performance.
+
 ### Accessibility
 
+Combobox is built with consideration for the [ARIA combobox pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/).
+
+When a user is focused on a Combobox, the listbox opens. When a user makes a selection, the listbox closes so the selected option can be registered to screen readers.
+
+#### Keyboard interaction
+
 When the user is focused on a Combobox, the following keyboard interactions apply:
 
 - Up and down arrows move the user between the options
 - Enter selects the currently active option
 
```

Comment:
todo: Add to the bottom of the list:

`- Escape closes the listbox or clears the selected option if the listbox is already closed`